config/routes.rb

```diff
@@ -357,7 +357,6 @@
     get "/async_info/shell_version", controller: "async_info#shell_version", defaults: { format: :json }
 
     get "/future", to: redirect("devteam/the-future-of-dev-160n")
-    get "/forem", to: redirect("devteam/for-empowering-community-2k6h")
```

Comment:
Note: the post or even pre deployment task is important to make sure this still works on DEV